core/zoe/contracts/simpleOffer/srcs/secondPriceSrcs.js

```diff
@@ -0,0 +1,134 @@
+import harden from '@agoric/harden';
+
+import { makeHasOkRules } from '../../../contractUtils';
+
+// The issuer array is ordered such that the item issuer is first
+// and the price issuer is second. All of the related arrays
+// (assays, strategies, etc.) also use this same ordering.
+const ITEM_INDEX = 0;
+const PRICE_INDEX = 1;
+
+// We expect the first offer to be the creator of the auction
```

Comment:
I have a version of the swap with invites (not in a PR yet since I was trying to land uniswap first, but it's the one we went over on the big screen), but I was hoping to show a usable version without invites, so that `simpleOffer` is really as simple as possible. 

I think you're 100% right on not relying on the order of offer to indicate position. Let me see if I can improve that. It may be that the non-invite model just doesn't work, but since it is so much simpler I'd like to prove that to ourselves. 

One difference in how we may be thinking about it: I wasn't envisioning that an Arthur the auctioneer existed, since it's trivial for anyone who wants to sell something at auction to make the auction themselves. I'm not sure what role Arthur would play or how he would improve anything. In the tests, Alice wants to sell something at auction, so she creates the auction instance and (keeping the auction reference to herself at first) puts the item up for auction. Then, only when she is assured that the item is up at auction does she spread the auction instance around to potential bidders. 

There's still the question of how, without invites, the potential bidders find the `zoeInstance` credible. That needs more work.